recipes/mingw-w64/all/conanfile.py

```diff
@@ -13,29 +13,193 @@ class MingwConan(ConanFile):
+            conf_args = []
+            autotools.configure(args=conf_args, target=target_tag, host=False, build=False)
+            autotools.make()
+            autotools.install()
+
+        self.output.info("Building mingw-w64-headers ...")
+        with tools.chdir(os.path.join(self.source_folder, "mingw-w64", "mingw-w64-headers")):
+            autotools = AutoToolsBuildEnvironment(self)
+            conf_args = [
+                "--with-widl={}".format(os.path.join(self.package_folder, "bin")),
+                "--enable-sdk=all",
+                "--prefix={}".format(os.path.join(self.package_folder, target_tag)),
+                "--with-sysroot={}".format(self.package_folder),
+            ]
+            autotools.configure(args=conf_args, target=False, host=target_tag, build=False)
+            autotools.make()
+            autotools.install()
+            # Step 3) GCC requires the x86_64-w64-mingw32 directory be mirrored as a
+            # directory 'mingw' in the same root.  So, if using configure default
+            # /usr/local, type:
+            #     ln -s /usr/local/x86_64-w64-mingw32 /usr/local/mingw
+            #     or, for sysroot, type:
+            #     ln -s /mypath/x86_64-w64-mingw32 /mypath/mingw
+            self.run(f"ln -s {os.path.join(self.package_folder, target_tag)} {os.path.join(self.package_folder, 'mingw')}")
+            # Step 5) Symlink x86_64-w64-mingw32/lib directory as x86_64-w64-mingw32/lib64:
+            # ln -s /usr/local/x86_64-w64-mingw32/lib /usr/local/x86_64-w64-mingw32/lib64
+            # or, for sysroot:
+            #     ln -s /mypath/x86_64-w64-mingw32/lib /mypath/x86_64-w64-mingw32/lib64
+            self.run(f"ln -s {os.path.join(self.package_folder, target_tag, 'lib')} {os.path.join(self.package_folder, target_tag, 'lib64')}")
+
+        with tools.environment_append(env):
+            self.output.info("Building core gcc ...")
+            with tools.chdir(os.path.join(self.source_folder, "gcc")):
+                autotools_gcc = AutoToolsBuildEnvironment(self)
+                conf_args = [
+                    "--enable-targets=all",
+                    "--enable-languages=c,c++",
+                    "--with-sysroot={}".format(self.package_folder),
+                    "--disable-shared",
+                    "--with-gmp={}".format(self.package_folder),
+                    "--with-mpfr={}".format(self.package_folder),
+                    "--with-mpc={}".format(self.package_folder)
+                ]
+                autotools_gcc.configure(args=conf_args, target=target_tag, host=False, build=False)
+                autotools_gcc.make(target="all-gcc")
+                autotools_gcc.make(target="install-gcc")
+
+        self.output.info("Building mingw-w64-crt ...")
+        with tools.chdir(os.path.join(self.source_folder, "mingw-w64", "mingw-w64-crt")):
+            autotools = AutoToolsBuildEnvironment(self)
+            conf_args = [
```

Comment:
Does this honor `self.options.exception` and `self.options.threads`?

Reply:
I found two configure options which can be set on gcc. Added it already locally, will push the commits soon.